lib/models/BucketInfo.ts

```diff
@@ -377,6 +480,13 @@ export default class BucketInfo {
      */
     static deSerialize(stringBucket: string) {
         const obj = JSON.parse(stringBucket);
+        const capabilities: CapabilitiesInput = {
+            ...obj.capabilities,
+            VeeamSOSApi: {
+                ...obj.capabilities?.VeeamSOSApi,
+                CapacityInfo: this.convertCapacityToBigInt(obj.capabilities?.VeeamSOSApi?.CapacityInfo),
+            },
+        };
```

Comment:
should we "create" the fields, if not provided; or keep the "status-quo", as before?

```suggestion
        const capabilities: CapabilitiesInput = obj.capabilities && {
            ...obj.capabilities,
            VeeamSOSApi: obj.capabilities.VeeamSOSApi && {
                ...obj.capabilities.VeeamSOSApi,
                CapacityInfo: this.convertCapacityToBigInt(obj.capabilities.VeeamSOSApi.CapacityInfo),
            },
        };
```